test_scripts/Polices/Policy_Table_Update/027_ATF_P_TC_HMILvl_on_PTU_affected_in_FULL_or_LIMITED.lua

```diff
@@ -182,19 +180,15 @@ function Test:RegisterSecondApp()
       self.mobileSession2:ExpectResponse(correlationId2, { success = true })
       self.mobileSession2:ExpectNotification("OnPermissionsChange")
     end)
-end
-
-function Test:ActivateFirstApp()
-  commonSteps:ActivateAppInSpecificLevel(self, HMIAppID)
+  self.mobileSession2:ExpectNotification("OnHMIStatus",{ systemContext = "MAIN", hmiLevel = "NONE"})
```

Comment:
@dboltovskyi, agree with you, but in case we have 2 applications: one in FULL, one in LIMITED, SDL will choose randomly between them. That may fail verification that is done with function: testCasesForPolicyTable:updatePolicyInDifferentSessions(line 192) and PTU will not be done.
I have add comment in confluence that additional script should be created for LIMITED application.